blockchain/reactor.go

```diff
@@ -290,11 +295,11 @@ var _ = wire.RegisterInterface(
 
 // DecodeMessage decodes BlockchainMessage.
 // TODO: ensure that bz is completely read.
-func DecodeMessage(bz []byte) (msgType byte, msg BlockchainMessage, err error) {
+func DecodeMessage(bz []byte, maxSize int) (msgType byte, msg BlockchainMessage, err error) {
```

Comment:
What's the rationale for us changing this API?

Reply:
Because we need some way to get the size in there that doesn't make the size a global.  Another option is to make this a method on something - I might rather that, but at least this is clear. Regardless, we're discussing un-exporting these functions anyways

Reply:
Oh yes, I recall the unexporting mention, thanks for the reminder. Gotcha.